tardis/tests/tests_slow/conftest.py

```diff
@@ -87,3 +88,20 @@ def reference(request, reference_datadir):
         luminosity_density_lambda=
             u.Quantity(spectrum['luminosity_density_lambda'], 'erg / (Angstrom s)'))
     return ndarrays
+
+
+@pytest.mark.hookwrapper
+def pytest_runtest_makereport(item, call):
+    pytest_html = item.config.pluginmanager.getplugin('html')
+    outcome = yield
+    report = outcome.get_result()
+    extra = getattr(report, 'extra', [])
+    if report.when == 'call':
+        test_method_name = report.nodeid.split('::')[-1]
+        if test_method_name == "test_spectrum":
+            # Just making it work for this commit. Will be changes subsequently
+            with open(os.path.join("/tmp/plots", tardis.__githash__[0:7],
```

Comment:
don't use the png file as a workaround - directly encode to base64 binary thingie